packages/office-ui-fabric-react/src/components/Button/ActionButton/ActionButton.tsx

```diff
@@ -11,17 +11,25 @@ export class ActionButton extends BaseComponent<IButtonProps, {}> {
    * Tell BaseComponent to bypass resolution of componentRef.
    */
   protected _shouldUpdateComponentRef = false;
+  private _button: BaseButton;
 
   public render() {
     let { styles, theme } = this.props;
 
     return (
       <BaseButton
         { ...this.props }
+        ref={ button => { if (button) this._button = button; } }
```

Comment:
You definitely don't need to change action button. You can just pass componentRef into it, like you do in dropdown, and this will work fine.

See

https://codepen.io/slimemold/pen/RjzMap?editors=1111